src/alarmsettings.cpp

```diff
@@ -42,20 +42,19 @@
 #include <QDBusMessage>
 #include <QDBusPendingCallWatcher>
 
-#define TIMED_CONNECTION       QDBusConnection::systemBus()
 #define TIMED_SERVICE          "com.nokia.time"
 #define TIMED_PATH             "/com/nokia/time"
 #define CLOCK_APP              "nemoalarms"
 
-AlarmSettings::AlarmSettings(QObject* parent) :
-    QObject(parent),
-    m_snooze(-1),
-    m_ready(false),
-    m_daemon(new TimeDaemon(TIMED_SERVICE, TIMED_PATH, TIMED_CONNECTION, this))
+AlarmSettings::AlarmSettings(QObject* parent)
+    : QObject(parent)
+    , m_snooze(-1)
```

Comment:
Should these be initialized in alarmsettings.h too?

Reply:
Didn't think it was worthwhile as the amount of member variables is small and there's only a single ctor.